src/app/static/.eslintrc.js

```diff
@@ -16,15 +16,20 @@ module.exports = {
         "no-debugger": process.env.NODE_ENV === "production" ? "warn" : "off",
         "comma-dangle": ["error", "never"],
         indent: ["error", 4],
-        quotes: ["error", "double", { avoidEscape: true }]
+        quotes: ["error", "double", { avoidEscape: true }],
+        "arrow-body-style": "off",
```

Comment:
These two rules don't seem to work well with Typescript - `arrow-body-style` seems to interpret a return type as a funtion body without braces. `prefer-default-export`. And I don't think you can specify a type with default export, hence removing `prefer-default-export`.